docs/application/tizen-studio/rt-ide/getting-started/overview.md

```diff
@@ -3,26 +3,27 @@
 The Tizen Studio for RT allows you to easily develop and test your RT application:
 
 1. [Install the Tizen Studio for RT](install.md) on your computer.
-2. [Manage the project](create.md), including creating a new Tizen Studio for RT project, creating application resources, and building the project.
+2. [Manage the project](create.md), which includes creating a new Tizen Studio for RT project, creating application resources, and building the project.
 3. [Flash the project](flash.md) on the board.
 4. [Use the serial terminal](terminal.md) to communicate with the board.
 5. [Debug the project](debug.md) to avoid runtime problems.
 
 
 ## Prerequisites
 
-Check the following prerequisites before installing the Tizen Studio for RT:
+To install the Tizen Studio for RT the following prerequistes must be checked:
 
-- Java Development Kit (JDK) requirements
+- You must install JDK and OpenJDK 8, 9 or 10 to use the Tizen Studio.
 
-  You must install a JDK 8 or 9 to use the Tizen Studio. Do not install OpenJDK.
+  To install the appropriate JDK and OpenJDK version for your Ubuntu system, go to [Ubuntu](https://www.ubuntu.com/) and follow the detailed instructions.
```

Comment:
Please re-check this update. With this update, in this list there are 6 items like below.

 - You must install JDK and OpenJDK 8, 9 or 10 to use the Tizen Studio.
 - Operating system and hardware requirements:
 - Operating system and hardware requirements
 - RT platform requirements:
 - Package requirements for developing applications in Ubuntu:
 - Docker CE for Ubuntu:

I think it would be better to revert this update to the original version...
The first item is in sentence form while others are in noun form, and the 1st item's description is actually same with 2nd item's description. Also the 2dn item's title is same with 3rd item's.
I can't get what this update is for...